sim_integration: 'rk4' 
initial_joint_angles: [0, 0, 0]
initial_base_position: [0, 0]
initial_base_angle: 0
goal_position: [2.5, 0.5]
goal_angle: 0
